flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/relational/connection/JdbcConnectionPools.java

```diff
@@ -31,16 +33,17 @@ public class JdbcConnectionPools implements ConnectionPools<HikariDataSource, Jd
 
     private static JdbcConnectionPools instance;
     private final Map<ConnectionPoolId, HikariDataSource> pools = new HashMap<>();
-    private static JdbcConnectionPoolFactory jdbcConnectionPoolFactory;
+    private static final Map<String, JdbcConnectionPoolFactory> poolFactoryMap = new HashMap<>();
```

Comment:
Just noticed that this field should be in uppercase.

Reply:
+1